backend/data/build.gradle

```diff
@@ -208,6 +207,11 @@ dependencies {
     implementation libs.slackApiClient
     implementation libs.slackApiModelKotlinExtension
     implementation libs.slackApiClientKotlinExtension
+
+    /**
+     * HANDLEBARS
+     */
+    implementation('com.github.jknack:handlebars:4.4.0')
```

Comment:
This is not relevant.

Reply:
Stray code from my previous mix-up. Updated now